### ISSUE TITLE
StartupBootstrapper exception

### ISSUE THREAD

**Issue by vmendi** [Microsoft/ApplicationInsights-Home#231]
We are running an Azure Web App that we recently upgraded to .net Core 2.1 with Microsoft.ApplicationInsights.AspNetCore 2.3.0.

We getting this exception during startup:

```Hosting startup assembly exception  System.InvalidOperationException: Startup assembly StartupBootstrapper failed to execute. See the inner exception for more details. ---> System.IO.FileNotFoundException: Could not load file or assembly 'StartupBootstrapper, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.     at System.Reflection.RuntimeAssembly.nLoad(AssemblyName fileName, String codeBase, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, IntPtr ptrLoadContextBinder)     at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, IntPtr ptrLoadContextBinder)     at System.Reflection.Assembly.Load(AssemblyName assemblyRef)     at Microsoft.AspNetCore.Hosting.WebHostBuilder.BuildCommonServices(AggregateException& hostingStartupErrors)     --- End of inner exception stack trace --```

Any clue on how to fix it?

Thanks

**Comment by cijothomas** [comment]
This might be due to an issue in Web Apps infra itself. Confirming with the relevant folks.

**Comment by mcliment** [comment]
I suspect this issue is related to this one https://github.com/aspnet/Hosting/issues/1398

In our case was related to using the Application Insights extension with a self contained app. I referenced Insights in the project directly and deactivated the extension as recommended in the issue.

**Comment by clguimanMSFT** [comment]
Application Insights extension for Azure App Service 2.6.3 was released - this version supports ASP.NET Core 2.1. Please upgrade to this version.

**Comment by SergeyKanzhelev** [comment]
Please re-open if suggested solution didn't work

**Comment by vmendi** [comment]
After updating to 2.6.3 today, we are not getting the exception anymore.

Thanks!

**Comment by SergeyKanzhelev** [comment]
@joshmouch the suggestion here is to update to the latest extension version, not to uninstall it. Please try it.

**Comment by joshmouch** [comment]
I did do that, which hasn't helped yet.
However, I do note one oddity.
When you click on the Azure Port/App Insights/Performance/Profiler, in the "Linked Apps" tab on my staging app, I see two apps listed there... both the staging app and also the prod app (which is the one that I can't seem to be able to enable the Profiler Web Job on).  Could this be causing trouble having one app listed in two "Linked Apps" sections?  How do I remove one of these?  I wonder if it was added in the middle of a slot swap?
